Source/PeleLM.cpp

```diff
@@ -4775,8 +4783,8 @@ PeleLM::compute_differential_diffusion_fluxes (const MultiFab& S,
         }
         if (updateFluxReg)
         {
-          getAdvFluxReg().FineAdd(*flux[d],d,nspecies+1,RhoH,1,0.5*dt);
-          getViscFluxReg().FineAdd(*flux[d],d,nspecies+2,RhoH,1,0.5*dt);
+          getAdvFluxReg().FineAdd(*flux[d],d,nspecies+1,RhoH,1,-0.5*dt);
+          getViscFluxReg().FineAdd(*flux[d],d,nspecies+2,RhoH,1,-0.5*dt);
         }
```

Comment:
Incrementing with fine fluxes scaled by a negative .5*dt is counter to how this is done everywhere else in AMReX codes.  If the sign is indeed correct, we need to change the sign of the definition of the flux so that it represents the direction of physical transport of energy and then change the sign here to be the usual sign of these things.

Reply:
It's just that the SDC factor is negative here. We could setup a cleaner where we have an 'sdc_factor' that hide the fact that's it's negative. Might even make it easier to read ...

Reply:
Ahh, yeah, that's tricky, isn't it?  Damn.